### PR TITLE
Parse text from markdown

### PR DESCRIPTION
**Pull request by xufangt**

### CODE REVIEW

**Comment by PhilipMay** [comment]
Can you please add this line to the header of the 2 modified files:

`# Copyright (c) 2023 Fang Xu, Deutsche Telekom AG`

**Comment by PhilipMay** [comment]
Please also modify the README Licensing paragraph and add yourself.

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ If you don't want to install all dependencies, see
 
 Copyright (c) 2023 Philip May\
 Copyright (c) 2023 Philip May, Deutsche Telekom AG
```
Comment (PhilipMay) [inline README.md:38]: This needs a `\` at the end.

**Comment by xufangt** [comment]
`Command not found: black.`
 need to install black formatter.

**Comment by xufangt** [comment]
> @xufangt Some Ruff checks fail. This can often be fixed by running `make format`. If you want to check formatting localy just run `make check`.
Already fixed.

**Comment by PhilipMay** [comment]
> need to install black formatter.

Yes. This can be done by `make install` or `poetry lock && poetry install --all-extras`

**Comment by PhilipMay** [comment]
Because of license issues (in conjunction with Stackoverflow) I can not merge this. I am sorry @xufangt .

**Comment by PhilipMay** [comment]
Because of license issues (in conjunction with Stackoverflow) I can not merge this. I am sorry @xufangt .

**Pull request closed without merging**